packages/expo-application/src/Application.ts

```diff
@@ -119,15 +119,25 @@ export enum ApplicationReleaseType {
   APP_STORE = 5,
 }
 
-// @docsMissing
+// @needsAudit
+/**
+ * **iOS Only.** Gets the iOS application release type.
+ * @return Returns a promise which fulfills with [`ApplicationReleaseType`](#applicationreleasetype) enum.
```

Comment:
```suggestion
 * @return Returns a promise which fulfills with an [`ApplicationReleaseType`](#applicationreleasetype).
```

since applicationreleasetype is `@docsMissing`, does that mean this anchor link won't point anywhere? I think before we would be able to warn in CI if a dead link was added so it would be nice to be able to detect those

Reply:
The section and enum values are there (so the link will work), but there is no additional description for the type itself or the specific enum values:
<img width="532" alt="Screenshot 2021-04-06 at 17 24 36" src="https://user-images.githubusercontent.com/719641/113736089-1baac200-96fd-11eb-9095-c4414d22598c.png">